### PR TITLE
Bunch of small fixes/ code improvements cause code is kinda messy

### PR DESCRIPTION
**Pull request by The-Forecaster**

### CODE REVIEW

**Review by srgantmoomoo — COMMENTED** [review]
Why would you make that static?

File: src/main/java/me/srgantmoomoo/postman/impl/clickgui/front/HudEditor.java [hunk 1]
```diff
@@ -11,11 +11,12 @@
 public class HudEditor extends Module {
 	public BooleanSetting exitToClickGui = new BooleanSetting("exitToClickGui", this, true);
 	
+	private static ResourceLocation shader = new ResourceLocation("minecraft", "shaders/post/blur" + ".json");
```
Comment (srgantmoomoo) [inline src/main/java/me/srgantmoomoo/postman/impl/clickgui/front/HudEditor.java:14]: Why would u make this static?

File: src/main/java/me/srgantmoomoo/postman/framework/module/Module.java [hunk 2]
```diff
@@ -39,22 +38,22 @@ public Module(String name, String description, int key, Category category) {
 	
 	public void addSettings(Setting... settings) {
 		this.settings.addAll(Arrays.asList(settings));
-		this.settings.sort(Comparator.comparingInt(s -> s == keyCode ? 1 : 0));
+		this.settings.sort(Comparator.comparingInt(setting -> setting == this.keyCode ? 1 : 0));
 	}
 	
 	public String getDescription() {
-		return description;
+		return this.description;
```
Comment (srgantmoomoo) [inline src/main/java/me/srgantmoomoo/postman/framework/module/Module.java:45]: Ok this shit mad hard to do on my phone… anyways, why are you making everrything this.? The program already knows what is being referenced, theres i nothin else in this class using those names

File: src/main/java/me/srgantmoomoo/postman/backend/util/Discord.java [hunk 3]
```diff
@@ -18,7 +18,7 @@ public static void startRPC() {
         
         if(Minecraft.getMinecraft().player != null && Minecraft.getMinecraft().world != null) {
         	discordRichPresence.details = Minecraft.getMinecraft().player.getName();
-        }else discordRichPresence.details = "in da menus";
+        } else discordRichPresence.details = "in da menus";
```
Comment (srgantmoomoo) [inline src/main/java/me/srgantmoomoo/postman/backend/util/Discord.java:21]: I like my elses with no spaces bfore them :). A lot of the code is this project looks weird with weird whitespace cause of how my eclipse was configured but my else’s are not mistakes.

File: src/main/java/me/srgantmoomoo/postman/impl/clickgui/front/HudEditor.java (hunk 1 above)
Comment (The-Forecaster) [reply src/main/java/me/srgantmoomoo/postman/impl/clickgui/front/HudEditor.java:14]: cause the only reference is a static one

File: src/main/java/me/srgantmoomoo/postman/framework/module/Module.java (hunk 2 above)
Comment (The-Forecaster) [reply src/main/java/me/srgantmoomoo/postman/framework/module/Module.java:45]: cause oop

**Comment by srgantmoomoo** [comment]
sorry, its a decent pr. some of the things ur doing in it just don't make sense to me. i don't see the reason to rewrite something just to change the style of it,

**Comment by TwoThreeTwo** [comment]
Just a bunch of spaces and a few rearrangements, 😢

**Pull request closed without merging**